war/src/main/java/io/lumeer/engine/rest/OrganizationService.java

```diff
@@ -0,0 +1,371 @@
+      }
+   }
+
+   /**
+    * @param organizationId organization id
+    * @param userName user name
+    */
+   @DELETE
+   @Path("/{organizationId}/users/{userName}")
+   public void removeUserFromOrganization(final @PathParam("organizationId") String organizationId, final @PathParam("userName") String userName) {
+      if (organizationId == null || userName == null) {
+         throw new IllegalArgumentException();
+      }
+      organizationFacade.removeUserFromOrganization(organizationId, userName);
+   }
+
+   /**
+    * @param organizationId organization id
+    * @param userName user name
+    * @return list of roles for given user
+    */
+   @GET
+   @Path("/{organizationId}/users/{userName}/roles")
+   @Produces(MediaType.APPLICATION_JSON)
+   public List<String> readUserRoles(final @PathParam("organizationId") String organizationId, final @PathParam("userName") String userName) {
+      if (organizationId == null || userName == null) {
+         throw new IllegalArgumentException();
+      }
+      return organizationFacade.readUserRoles(organizationId, userName);
+   }
+
+   /**
+    * @param organizationId organization id
+    * @param userName user name
+    * @param role role to add
+    */
+   @POST
+   @Path("/{organizationId}/users/{userName}/roles/{role}")
+   @Consumes(MediaType.APPLICATION_JSON)
+   public void addRoleToUser(final @PathParam("organizationId") String organizationId, final @PathParam("userName") String userName, final @PathParam("role") String role) {
+      if (organizationId == null || userName == null || role == null) {
+         throw new IllegalArgumentException();
+      }
+      organizationFacade.addRolesToUser(organizationId, userName, Arrays.asList(role));
+   }
+
+   /**
+    * @param organizationId organization id
+    * @param userName user name
+    * @param role role to remove
+    */
+   @DELETE
+   @Path("/{organizationId}/users/{userName}/roles/{role}")
+   @Consumes(MediaType.APPLICATION_JSON)
+   public void removeRoleFromUser(final @PathParam("organizationId") String organizationId, final @PathParam("userName") String userName, final @PathParam("role") String role) {
+      if (organizationId == null || userName == null || role == null) {
+         throw new IllegalArgumentException();
+      }
+      organizationFacade.removeRolesFromUser(organizationId, userName, Arrays.asList(role));
+   }
+
+   /**
+    * @param organizationId organization id
+    * @return list of default roles
+    */
+   @GET
+   @Path("/{organizationId}/roles")
+   @Produces(MediaType.APPLICATION_JSON)
+   public List<String> readDefaultRoles(final @PathParam("organizationId") String organizationId) {
+      if (organizationId == null) {
+         throw new IllegalArgumentException();
+      }
+      return organizationFacade.readDefaultRoles(organizationId);
+   }
+
+   /**
+    * @param organizationId organization id
+    * @param userRoles list of default roles
+    */
+   @POST
```

Comment:
Does this method override all the default roles? If so, it should rather be PUT in my opinion.